src/univers/version_range.py

```diff
@@ -233,7 +233,8 @@ class NpmVersionRange(VersionRange):
         ">=": ">=",
         "<": "<",
         ">": ">",
-        "=": "=",  # This is not a native node-semver comparator, but is used in the gitlab version range for npm packages.
+        "=": "=",
+        # This is not a native node-semver comparator, but is used in the gitlab version range for npm packages.
```

Comment:
good catch, end of line was a bad idea of mine, but this comment should be above line 236, not after